### ISSUE TITLE
How to deal with high contrast background colors?

### ISSUE THREAD

**Issue by erik1110** [vanhuyz/CycleGAN-TensorFlow#124]
I want to do gender transformation so I have collected men and women pictures which each has about 3000.
Now I get starting training at step 1800. I take a look at tensorborad at IMAGES pages, and the generated images get high contrast background colors.

The author said "If high contrast background colors between input and generated images are observed (e.g. black becomes white), you should restart your training!" in Readme. I have restart it many times, but I still get high contrast background colors. How do I solve this problem? 

The loss is belowed.
<img width="1241" alt="螢幕快照 2020-11-14 下午4 58 49" src="https://user-images.githubusercontent.com/40282726/99143734-c6610c00-269a-11eb-83bf-50d04f07841b.png">
